kbase-extension/static/kbase/css/appCell.css

```diff
@@ -29,66 +29,7 @@
 }
 
 .kb-app-cell .kb-app-parameter-row:hover {
-  background-color: rgba(0, 0, 0, 0.03);
-}
-
-.kb-app-row-close-btn-addon {
-  background: transparent;
-  border: 0;
-}
-
-.kb-app-row-clip-btn-addon {
-  background: transparent;
-  border: 0;
-  padding: 0;
-  padding-right: 10px;
-  height: 100%;
-}
-
-.kb-app-row-clip-btn-addon:active::after {
-  background-color: #fff;
-  border-radius: 3px;
-  border: 1px solid gray;
-  content: 'copied!';
-  margin-top: -5px;
-  padding: 7px;
-  position: absolute;
-}
-
-.kb-input-group-wide {
-  width: 100%;
-}
-
-.kb-input-row-flex {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-}
-
-.cell.selected .btn-default.kb-app-row-close-btn {
-  color: gray;
-}
-
-.kb-app-row-clip-btn,
-.kb-app-row-close-btn {
-  height: 100%;
-  margin-left: 1px;
-  border-right: 0;
-  border-top: 0;
-  border-bottom: 0;
-  border: none;
-  background-color: transparent;
-  color: gray;
-}
-
-/* let's just do this as a normal button hover effect */
-
-.kb-app-parameter-right-error-bar {
-  height: 28px;
-
-  /* firefox wants it explicitly wired */
-
-  background: red;
+  background-color: rgb(0 0 0 / 0.03);
```

Comment:
I did some reordering in this file, mostly to move all the `.kb-app-cell` declarations to be next to each other to make life easier when it gets converted to scss.

Reply:
stylelint removes unneeded commas in `rgb` values